rocAL/rocAL/source/augmentations/color_augmentations/node_brightness.cpp

```diff
@@ -25,43 +25,43 @@ THE SOFTWARE.
 #include "exception.h"
 
 
-BrightnessNode::BrightnessNode(const std::vector<Image *> &inputs, const std::vector<Image *> &outputs) :
+BrightnessNode::BrightnessNode(const std::vector<rocalTensor *> &inputs, const std::vector<rocalTensor *> &outputs) :
         Node(inputs, outputs),
         _alpha(ALPHA_RANGE[0], ALPHA_RANGE[1]),
-        _beta (BETA_RANGE[0], BETA_RANGE[1])
-{
-}
+        _beta (BETA_RANGE[0], BETA_RANGE[1]) { }
 
-void BrightnessNode::create_node()
-{
+void BrightnessNode::create_node() {
     if(_node)
         return;
 
     _alpha.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
     _beta.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
 
-    _node = vxExtrppNode_BrightnessbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), _batch_size);/*A temporary fix for time being*/
+    int input_layout = (int)_inputs[0]->info().layout();
+    int output_layout = (int)_outputs[0]->info().layout();
+    int roi_type = (int)_inputs[0]->info().roi_type();
+    vx_scalar in_layout_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &input_layout);
+    vx_scalar out_layout_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &output_layout);
+    vx_scalar roi_type_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &roi_type);
+    _node = vxExtrppNode_Brightness(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), in_layout_vx, out_layout_vx, roi_type_vx, _batch_size);
```

Comment:
alpha and beta needs to be passed through a single tensor instead of 2 arrays and remove batch_size